src/test/js/multibranch/multibranchOpening.js

```diff
@@ -0,0 +1,72 @@
+/** @module multibranchOpening
+ * @memberof multibranch
+ * @description TEST: basic tests around multiBranch and encoding of branches
+ */
+const async = require('async');
+const path = require('path');
+const pathToRepo = path.resolve('./target/test2-project-folder');
+const sourceRep = './src/test/resources/multibranch_2';
+const git = require('../../../main/js/api/git');
+const pageHelper = require('../../../main/js/util/pageHelper');
+const createCallbackWrapper = pageHelper.createCallbackWrapper;
+const sanityCheck = pageHelper.sanityCheck;
+
+module.exports = {
+  // ** creating a git repo */
+  before: function (browser, done) {
+    // we creating a git repo in target based on the src repo (see above)
+    git.createRepo(sourceRep, pathToRepo)
+      .then(function () {
+        git.createBranch('feature/1', pathToRepo)
+          .then(done);
+      });
+  },
+  /**
+   * Make sure we can open the master and the feature/1 branch results screen from activity and branch tab
+   * Regression: https://issues.jenkins-ci.org/browse/JENKINS-40027
+   */
+  'open master and feature/1 branch from activity and from branch tab': function (browser) {
+    const cases = [{
```

Comment:
this adds very little to the test. I am only using the partial id matching as a way to find a branch name in the list, but this elevates it to a higher level without actually explaining what the patterns are, this is less clear than before.